src/kubernetes/state/evmap.rs

```diff
@@ -35,9 +40,12 @@ where
         // Prepare flush debounce.
         let debounced_flush = flush_debounce_timeout.map(Debounce::new);
 
+        let hash_key = key;
+
         Self {
             inner,
             debounced_flush,
+            hash_key,
```

Comment:
You could just name the parameter `hash_key` to start off with to avoid the extra `let`, or here you could do:

```
hash_key: key
```

I think having that extra `let` there is a little confusing.